src/conspiracies/docprocessing/coref/coref_component.py

```diff
@@ -174,6 +176,7 @@ def __call__(self, doc: Doc) -> Doc:
     "allennlp_coref",
     default_config={
         "model_path": None,
+        "language": "da",
```

Comment:
I would give the option to specify model_path, but otherwise just fetch the language from the nlp object.

Reply:
You _can_ specify model path. But getting language from nlp object is a good idea. That did not occur to me.